src/components/CardActions.tsx

```diff
@@ -0,0 +1,77 @@
+import { CardAttributes } from "@/types/interfaces";
+
+interface CardActionsProps {
+  card: CardAttributes;
+}
+
+export default function CardActions({ card }: CardActionsProps) {
+  const handleCopyRawText = () => {
+    navigator.clipboard.writeText(card.description_raw).then(() => {
```

Comment:
Always wondered how these buttons work since we don't do anything with the window or navigator on the back-end obviously.  Super cool what different languages have built in.